pkg/session_recording/recordings.jsx

```diff
@@ -395,16 +397,13 @@
 
             for (i = 0; i < entryList.length; i++) {
                 let e = entryList[i];
-                let boot_id = e["_BOOT_ID"];
-                let session_id = e["TLOG_SESSION"];
-                let process_id = e["_PID"];
+                let id = e['TLOG_REC'];
 
                 /* Skip entries with missing session ID */
-                if (session_id === undefined) {
+                if (id === undefined) {
                     continue;
                 }
```

Comment:
This had sense for compatibility with older tlog version which wasn't adding the extra journal fields. It doesn't make such a good sense now in this specific shape, but we still have to ignore the log messages which don't have the extra fields added. And indeed checking if TLOG_REC is set might be good for that. Only please change the comment.